pyproject.toml

```diff
@@ -0,0 +1,73 @@
+[build-system]
+requires = ["poetry-core>=1.8"]
+build-backend = "poetry.core.masonry.api"
+
+[tool.poetry]
+name = "hightime"
+version = "0.3.0-dev0"
```

Comment:
Note that I bumped the version from 0.2.2 to 0.3.0-dev0. Since major<1, this is considered a semver break.

I checked codesearch for `hightime\s*=\s*"\^ file:pyproject\.toml$` and it looks like I need to update nidaqmx-python.